test/oli_web/live/delivery/student_dashboard/components/actions_tab_test.exs

```diff
@@ -109,6 +111,59 @@ defmodule OliWeb.Delivery.StudentDashboard.Components.ActionsTabTest do
                "Actions"
              )
     end
+
+    test "can unenroll students", %{
+      section: section,
+      conn: conn,
+      instructor: instructor
+    } do
+      student = insert(:user)
+      Sections.enroll(instructor.id, section.id, [ContextRoles.get_role(:context_instructor)])
+      Sections.enroll(student.id, section.id, [ContextRoles.get_role(:context_learner)])
+
+      {:ok, view, _html} =
+        live(conn, live_view_students_actions_route(section.slug, student.id, :actions))
+
+      # Actions tab is the selected one
+      assert has_element?(
+               view,
+               ~s{a[href="#{live_view_students_actions_route(section.slug, student.id, :actions)}"].border-b-2},
+               "Actions"
+             )
+
+      assert has_element?(view, "button", "Unenroll")
+      assert has_element?(view, "select[name=filter_by_role_id]")
+
+      view
+      |> with_target("#unenroll_user_modal")
+      |> render_click("open", %{})
+
+      assert has_element?(
+               view,
+               "p",
+               "Are you sure you want to unenroll \"#{student.name}\" from the course \"#{section.title}\"?"
+             )
+
+      view
+      |> with_target("#student_actions")
+      |> render_click("unenroll", %{})
+
+      assert_redirected(
+        view,
+        Routes.live_path(
+          OliWeb.Endpoint,
+          OliWeb.Delivery.InstructorDashboard.InstructorDashboardLive,
+          section.slug,
+          :manage
+        )
+      )
+
+      assert Enrollment
+             |> where([e], e.user_id == ^student.id and e.section_id == ^section.id)
+             |> limit(1)
+             |> Oli.Repo.one()
+             |> Map.get(:status) == :suspended
```

Comment:
I'm not sure, but maybe this piece of test could be moved to the `sections_test.exs` module where there are some tests referring to enroll and unenroll actions (I don't know if this has been tested yet).

Reply:
As discussed offline, I'm only querying the database to see if the user interaction actually made the user unenrolled, so it's ok for this to be here 😁